docs/examples/scrollbar/manual-scroll.vue

```diff
@@ -17,26 +17,27 @@
 
 <script lang="ts" setup>
 import { onMounted, ref } from 'vue'
-import { ElScrollbar } from 'element-plus'
+import { isNumber } from 'lodash-unified'
```

Comment:
To be compatible with playground, we should not import lodash-unifiend, we can use typeof directly.